### ISSUE TITLE
Not able to set `aria-valuetext` on slider thumb

### ISSUE THREAD

**Issue by Atlasfreak** [mantinedev/mantine#7214]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.13.5

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Not applicable – issue is not related to the browser

### Describe the bug

The optional aria attribute `aria-valuetext` can't be correctly set for the slider as it is not passed to the thumb and only applied to the parent element.
See [mdn docs on slider role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role) for more information.

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/sandbox/mantine-react-template-forked-lnvhdt

### Possible fix

I can see some possible fixes one would be to pass the `aria-valuetext` attribute down to the thumb element. Another would be to create seperate attribute `thumb-valuetext` (or similar) and use that to populate the `aria-valuetext` in the thumb element.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by onkar-deshpande** [comment]
Option `aria-valuetext` should be present when slider doesn't contain numeric values for `aria-valuenow`. However, if you can see the in attached sandbox, `aria-valuenow` has numeric value so no need to have optional attribute `aria-valuetext` as per docs.

**Comment by Atlasfreak** [comment]
> Option `aria-valuetext` should be present when slider doesn't contain numeric values for `aria-valuenow`. However, if you can see the in attached sandbox, `aria-valuenow` has numeric value so no need to have optional attribute `aria-valuetext` as per docs.

As you can see in the example provided by mdn it is indeed very useful for sliders with numeric values. Espacially to give more meaning to the numeric values so screen readers don't just read the value but also the unit of the value for example.

**Comment by rtivital** [comment]
Starting from 7.15.1, you can pass props down to thumb with `thumbProps={{ 'aria-valuetext': 'Your text' }}`

**Comment by rtivital** [comment]
Fixed in [7.15.1](https://github.com/mantinedev/mantine/releases/tag/7.15.1)
